release_util/management/commands/generate_history.py

```diff
@@ -0,0 +1,136 @@
+"""
+Management command to populate initial history.
+"""
+import logging
+import time
+from datetime import datetime
+from django.apps import apps
+from django.core.management.base import BaseCommand
+from django.db import connection, transaction
+
+get_model = apps.get_model
+
+log = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    Populate initial history for models using django-simple-history.
+    Example usage:
+    $ ./manage.py lms generate_history --models organizations.organization entitlements.courseentitlement --batchsize 1000 --sleep_between 1 --settings=devstack
+    """
+
+    help = (
+        "Populates the corresponding historical records with"
+        "the current state of records which do not have a historical record yet"
```

Comment:
Nit: This `help` should likely also call out django-simple-history explicitly, as the docstring does.